docs/getting-started/getting-started-with-wdl.rst

```diff
@@ -208,11 +208,56 @@ the Dockstore command line and descriptor rather than just directly
 calling it via Docker. This will test that the WDL correctly describes
 how to run your tool.
 
+Setting Up the Dockstore CLI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Line 212's length should match line 211's to be consistent with other files in the project.